catalyst_voices/lib/common/bip39/seed_phrase.dart

```diff
@@ -0,0 +1,82 @@
+import 'dart:typed_data';
+import 'package:bip39/bip39.dart' as bip39;
+
+/// Represents a seed phrase consisting of a mnemonic and provides methods for
+/// generating and deriving cryptographic data from the mnemonic.
+///
+/// The `SeedPhrase` class allows creation of a seed phrase either randomly,
+/// from a given mnemonic, or from entropy data. It supports converting between
+/// different formats, including Uint8List and hex strings.
+class SeedPhrase {
```

Comment:
This class should belong to catalyst_voices_models package. It's a place where different data structures, entities, models, etc. live.